parser/src/main/kotlin/com/guerinet/mymartlet/model/Statement.kt

```diff
@@ -25,4 +25,4 @@ data class Statement(
     val date: LocalDate,
     val dueDate: LocalDate,
     val amount: Double
-)
```

Comment:
Re: https://github.com/diffplug/spotless/issues/359

Note the "no-newline" on the left, while it's missing on the right.  That's because the file on the left is missing a newline, but the file on the right has it.

Reply:
Hm you're right. I think this was a false alarm, as we thought no new lines existed from the github preview. They all exist locally, so I will close that issue